packages/web-components/src/switch/switch.ts

```diff
@@ -0,0 +1,21 @@
+import { attr, booleanConverter, FASTElement } from '@microsoft/fast-element';
+
+export class Switch extends FASTElement {
```

Comment:
Is there a reason not to extend the FASTSwitch which provides Form Association and a11y all up? For the most part, I think what we end up having to manage given what I see here is styles for switch.

Reply:
Here's the full class we'll end up extending here for reference: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/switch/switch.ts

If there were "fluent" specific implementations around appearance we would likely add to the below class. As it stands now, I think it likely can stay empty by leveraging the above.
```ts
export class Switch extends FASTSwitch {}
```

Reply:
> Is there a reason not to extend the FASTSwitch which provides Form Association and a11y all up? For the most part, I think what we end up having to manage given what I see here is styles for switch.

No particular reason.. when we started the switch in our horizons repo it was suggested to me to start with the native checkbox rather than extending from FAST. But I definitely agree with you here that it makes sense to extend the FASTSwitch.

Reply:
Gotcha - Just as a reminder, our current working model is to start w/ FAST Foundation as an accelerator and align where need be first. Happy to help with any implementation questions you have!